sdk/java-sdk/src/main/java/kalix/javasdk/DeferredCallResponseException.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.javasdk;
+
+/**
+ * Exception used when a DeferredCall fails to wrap the origin error, plus the error code associated.
+ */
+public interface DeferredCallResponseException {
```

Comment:
Not sure we need an interface. We could have a case class exception directly here.

Reply:
update now